rdbishop19/Debug-Docket
# Auth hooks

# Description

Authorization using React Hooks. User can login or register a new account successfully.

Fixes #7 
Fixes #2 
Fixes #1 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

`git fetch --all`
`git checkout auth-hooks`
Open the app and go to the `/login` or `/register` routes.
In both, you should see a link to be able to switch between the two forms.
Try creating a new user or signing in with a previous user.
If you check the `Remember me?` checkbox, the storage will switch between local and session storage

# Checklist:
